specification/maps/data-plane/Creator/preview/2022-01-01-preview/features.json

```diff
@@ -0,0 +1,964 @@
+                "name": "GeoJsonGeometryCollection"
+              }
+            ]
+          }
+        },
+        "featureTypes": {
+          "description": "type of features returned",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "properties": {
+          "description": "attributes of the collection used",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DefinitionProperty"
+          }
+        },
+        "links": {
+          "description": "Links to other Features endpoints.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/FeaturesEndpointLink"
+          }
+        }
+      }
+    },
+    "DefinitionProperty": {
+      "type": "object",
+      "required": [
+        "name",
+        "required",
+        "type"
+      ],
+      "properties": {
+        "name": {
+          "description": "name of attribute",
+          "type": "string",
+          "example": "external_id"
+        },
+        "required": {
+          "description": "is attribute required",
+          "type": "boolean"
+        },
+        "type": {
+          "description": "type of attribute",
+          "type": "object"
+        }
+      }
+    },
+    "ExtendedGeoJsonFeatureCollection": {
+      "description": "A valid `GeoJSON FeatureCollection` object type extended with numberReturned and links array. Please refer to [RFC 7946](https://tools.ietf.org/html/rfc7946#section-3.3) for details.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../Common/preview/1.0/common.json#/definitions/GeoJsonFeatureCollectionData"
+        },
+        {
+          "$ref": "#/definitions/ExtendedGeoJsonFeatureCollectionData"
+        }
+      ],
+      "properties": {
+        "type": {
+          "description": "Specifies the `GeoJSON` type: FeatureCollection.",
+          "type": "string",
+          "enum": [
+            "FeatureCollection"
+          ],
+          "x-ms-enum": {
+            "name": "GeoJsonObjectType",
+            "modelAsString": false,
+            "values": [
+              {
+                "value": "FeatureCollection",
+                "description": "`GeoJSON FeatureCollection` object."
+              }
+            ]
+          }
+        }
+      }
+    },
+    "ExtendedGeoJsonFeatureCollectionData": {
+      "type": "object",
+      "properties": {
+        "ontology": {
+          "$ref": "../../../Common/preview/1.0/common.json#/definitions/Ontology"
+        },
+        "numberReturned": {
+          "description": "Number of returned features.",
+          "type": "integer"
+        },
+        "links": {
+          "description": "Links to other Features endpoints.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/FeaturesEndpointLink"
+          }
+        }
+      }
+    },
+    "FeatureResult": {
```

Comment:
add azuremapsintid and featureid ?

Reply:
Good question! If I'm not mistaken, azuremapsintid and featureid fall under the `properties` field of a feature which is already defined as part of `/common.json#/definitions/GeoJsonFeature`, so I don't think we need to specifically single out those properties.